docs/Operators.md

```diff
@@ -13975,6 +13976,128 @@ expect(node, inputs=[data, indices, updates], outputs=[y],
 </details>
 
 
+### <a name="ScatterND"></a><a name="scatternd">**ScatterND**</a>
+
+  ScatterND takes three inputs `data` tensor of rank r >= 1, `indices` tensor of rank q >= 1,
+  and `updates` tensor of rank q + r - indices.shape[-1] - 1. The output of the operation
+  is produced by creating a copy of the input `data`, and then updating its value to values
+  specified by `updates` at specific index positions specified by `indices`. Its output shape
+  is the same as the shape of `data`. Note that `indices` should not have duplicate entries.
+  That is, two or more `updates` for the same index-location is not supported.
+  
+  `indices` is an integer tensor. Let k denote indices.shape[-1], the last dimension in the shape of `indices`.
+   `indices' is treated as a (q-1)-dimensional tensor of k-tuples, where each k-tuple is a partial-index into `data`.
```

Comment:
`indices' <- closing quote should be backtick

Reply:
Thanks!